content/index.hbs

```diff
@@ -65,18 +66,26 @@
 					</article>
 				</div>
 				<div class="column">
-					<img src="img/ftp-group.jpg" alt="AS 200s conduct drill practice" />
+					<img src="img/det-flag.jpg" alt="AS 300 by det flag" />
 					<article>
 						<h2>How do I enroll?</h2>
 						<p>Ready for the challenge?</p>
 						<p>Enrollment information for next semester can be found at the link below.</p>
 						<a href="join.html">Enrollment <span class="icon"><i class="fas fa-arrow-right"></i></span></a>
 					</article>
 				</div>
+				<div class="column">
+					<img src="img/ussf.jpg" alt="Logo of the USSF" />
+					<article>
+						<h2>Interested in the Space Force?</h2>

```

Comment:
This seems like an odd place to put this GT article link. I think we'd be better off putting this into a new year on the timeline on the about page.

Reply:
I see what you mean. I will work on that